src/builder.js

```diff
@@ -553,6 +558,55 @@ export default function builder() {
       };
     });
 
+    instrument(window, 'onunhandledrejection', function(original) {
+      // See https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event
+      function onunhandledrejection(promiseRejectionEvent) {
+        debug('window.onunhandledrejection callback invoked', arguments);
+
+        // Skip if the error is already being sent.
+        if (currentErr) { return; }
+
+        if (!onUnhandledRejectionEnabled()) { return; }
+
+        let { reason } = promiseRejectionEvent
+        
+        // This error message is copied from the Node.js warning
+        // See https://nodejs.org/api/process.html#process_event_unhandledrejection
+        const PROMISE_REJECTION_WARNING = 'UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().'
+
+        if (reason instanceof Error) {
+          // simulate v8 stack
+          let fileName = reason.fileName || 'unknown'
+          let lineNumber = reason.lineNumber || 0
+          let stackFallback = `${reason.message}\n    at ? (${fileName}:${lineNumber})`
+
+          let stack = stackTrace(reason) || stackFallback
+
+          notify(reason, {
+            message: `UnhandledPromiseRejectionWarning: ${reason}`,  
+            stack: stack + "\n" + PROMISE_REJECTION_WARNING
+          });
+
+          return;
+        }
+
+        let message = typeof reason === 'string' ? reason : JSON.stringify(reason)
+        notify({
+          name: 'window.onunhandledrejection',
+          message: `UnhandledPromiseRejectionWarning: ${message}`,
+          stack: PROMISE_REJECTION_WARNING
+        });
+      }
+
+      return function(promiseRejectionEvent) {
+        onunhandledrejection(promiseRejectionEvent);
+        if (typeof original === 'function') {
+          return original.apply(this, arguments);
+        }
+        return false;
```

Comment:
Do you know if the return value here has any significance? I can't remember.

Reply:
I was following the pattern from the `onerror` callback. Looking back at the docs on MDN, it seems that the return value of `onerror` is significant, but the return value of `` is not. I think I will remove that.

https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onunhandledrejection

Reply:
Removed the return value.